docs/vignette_parallel_phylo.md

```diff
@@ -0,0 +1,102 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.9.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+```{code-cell}
+:tags: [remove-cell]
+
+import pyslim, tskit, msprime
+from IPython.display import SVG
+import numpy as np
+import pandas as pd 
+import os
+import util
+
+np.random.seed(1234)
+```
+
+
+(sec_vignette_parallel)=
+
+
+# Vignette: Parallelizing SLiM simulations in a phylogenetic tree
+
+Imagine you want to simulate the evolutionary history of a group. If there is
+no migration between any of the branches in your tree, any branches stemming
+from the same node can be simulated in parallel (see {numref}`phylo`).
+
+```{figure} _static/phylo.pdf
+:height: 200px
+:name: phylo
+
+Example of phylogeny we might want to simulate. Note how branches with the same color can be simulated in parallel when there is no migration.
+```
+
+## Simulating the branches
+First, we need to write a SLiM script that will be used for simulating the
+history of each branch in our phylogeny.
+We will perform a simple simulation, in which each branch can have a different
+(but fixed) population size and length (number of generations).
+Also, we will allow deleterious mutations to happen across the entire chromosome
+at a fixed rate.
+
+Here is a SLiM script that would do this:
+
+```{literalinclude} phylo_bgs.slim
+```
+
+For each branch, we will tell SLiM whether we want to start it from a previous
+branch or not, read the previous tree sequence (if necessary) and change the
+population size accordingly.
+Note that when you read a tree sequence into SLiM, the generation counter will
+be updated with the time encoded in the tree sequence, so we need to set the end
+of the simulation as the length of the branch (`gens`) plus the total length of the
+loaded tree sequence.
+At the end of the simulation, we call `sim.treeSeqRememberIndividuals` right 
+before saving the resulting tree sequence. This is necessary because we need to
+ensure the individuals in the final generation are never dropped from the tree 
+sequence in simplification, as they will later be used to glue the tree sequences
```

Comment:
maybe instead of `in simplification`, say `in future runs of SLiM which are started using the output of the simulation` or something